2.1.7

elzadj opened the pull request:
tests for now

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Enhanced icon control with responsive stroke/fill/outline and link-only typography mode.
  * Improved scroll behavior for in-page anchor navigation.

* **UI/UX Improvements**
  * Background options now show text labels (None, Solid, Gradient).
  * Visual refinements: alignment, spacing, inspector controls, and updated icon color token.

* **Bug Fixes**
  * Corrected category filtering for dynamic content queries and various styling/layout issues.

* **Localization**
  * Updated translation strings and POT metadata.

* **Tests**
  * Stabilized and modernized end-to-end tests for dynamic content and controls.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

elzadj committed:
Fix show hide breaking relations

elzadj committed:
revert file

elzadj committed:
Fix pagination not showing when a post is assigned to the child category only

elzadj committed:
Fix ib when using the default block style

elzadj committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into show-hide-breakpoint

elzadj committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into show-hide-breakpoint

elzadj committed:
fix most of the tests

elzadj committed:
Fix rest of DC tests

elzadj committed:
better dc media test and remove additional alignment tab for text block

elzadj committed:
update packages, fix DC toolbar styles

elzadj committed:
remove old alignment control test

elzadj committed:
Fix other toolbar fropdown styles

elzadj committed:
Remove not-needed tests

elzadj committed:
Rewrite dc post test

elzadj committed:
Fix 6.9 removing Text block if it set as default

elzadj committed:
Fix gutenberg toolbar randomly showing for maxi blocks on WP 6.9

elzadj committed:
Fix 400 errors for pagination on backend

elzadj committed:
Fix DC tests

elzadj committed:
Fix DC tests

elzadj committed:
improve column pattern test

elzadj committed:
Merge branch 'wp-6-9-fixes' of https://github.com/maxi-blocks/maxi-blocks into 2.1.7

elzadj committed:
change version

kyrapieterse committed:
Update tab controls to use text labels and improve styles

Replaces icon-only background and icon tab options with text labels ('None', 'Solid', 'Gradient') for better clarity. Refines SCSS for background control buttons, adding unique classes and selected states for each type. Also updates selected tab colour in setting-tabs-control to use the primary colour.

kyrapieterse committed:
Update border color in editor.scss

Changed the border color of the style control from var(--maxi-mid-green) to var(--maxi-grey-light) for improved visual consistency.

kyrapieterse committed:
Comment out tab button styles in editor.scss

The styles for .maxi-tabs-control__button-image, .maxi-tabs-control__button-video, and .maxi-tabs-control__button-svg have been commented out. This may be to temporarily disable these styles or for future refactoring.

kyrapieterse committed:
Refactor background tab button styles

Consolidated the styles for 'none', 'solid', and 'gradient' background type buttons into a single rule to reduce duplication and improve maintainability.

elzadj committed:
Fix show hide on hover for higher breakpoitns

elzadj committed:
Merge branch 'show-hide-breakpoint' of https://github.com/maxi-blocks/maxi-blocks into 2.1.7

elzadj committed:
Merge branch 'Remove-icons-from-background-button---ui-20-11-29025' of https://github.com/maxi-blocks/maxi-blocks into 2.1.7

elzadj committed:
Fix svg icon bg for button test

elzadj committed:
Fix anchors breaking the scroll effect

kyrapieterse committed:
Add detailed comments to IconControl component

Enhanced the IconControl and IconControlResponsiveSettings components with comprehensive inline comments and JSDoc blocks. These additions clarify the purpose, key features, and logic of each section, improving maintainability and developer understanding without altering functionality.

kyrapieterse committed:
Remove unnecessary <hr /> from icon-only toggle

Eliminated the horizontal rule before the icon-only toggle in IconControlResponsiveSettings to clean up the UI.

kyrapieterse commented:
<img width="512" height="314" alt="image" src="https://github.com/user-attachments/assets/46ada6a5-6210-49d2-a5ea-e1a481a6850d" />
missing links settings

elzadj committed:
Return back the tab in link control

kyrapieterse committed:
Show inherit toggle only for color icon style

Moved the inherit color toggle to display only when the icon style is 'color', preventing it from appearing for other styles. This improves the UI by ensuring the toggle is only shown in relevant contexts.

elzadj committed:
attempts to fix scrolling effects

kyrapieterse committed:
Show stroke width control only for color icon style

Moved the SvgStrokeWidthControl to render only when iconStyle is 'color' and svgType is not 'Shape'. This ensures the stroke width control is not shown for other icon styles, improving UI consistency.

Olekrut commented:
> 1. about [Anchor link bug #6080](https://github.com/maxi-blocks/maxi-blocks/issues/6080)

Fixed as well

kyrapieterse committed:
Show border controls only for 'border' icon style

Updated the IconControl component to display border controls exclusively when the icon style is set to 'border', removing the previous condition that also checked for 'button-block' in inlineTarget.

kyrapieterse committed:
Remove unnecessary padding from icon control field

Eliminated the forced top padding from .maxi-icon-control__inherit .maxi-base-control__field in the editor SCSS to improve layout consistency.

kyrapieterse committed:
Improve alignment and display control styles

Adjusted button width and margin in alignment control for better spacing, added a specific label class for alignment control, and updated the label class usage in the component. Enhanced display control styles to highlight selected list view rows and update text/icon color for better visibility.

kyrapieterse committed:
Add custom class and style for background control

Introduces a dynamic className to the background control in inspector-background.js and adds corresponding SCSS for custom margin on the gradient picker UI. This improves styling flexibility for background controls in the button-maxi block.

kyrapieterse committed:
Refactor icon control to use dynamic prop keys

Replaces direct usage of template string keys in destructuring with variables for icon-only, icon-inherit, and icon-content props. Adds effect to switch icon style based on SVG type and breakpoint, improving maintainability and handling of icon style state.

kyrapieterse reviewed (approved):
Looks good. Well done 🤘

elzadj committed:
Merge branch 'Button-Maxi' of https://github.com/maxi-blocks/maxi-blocks into 2.1.7

elzadj committed:
Return button icon bg inherit toggle

elzadj committed:
Smaller info box and fix bg gradient UI

elzadj committed:
Rewrite button icon control test

elzadj committed:
remove not-needed patch file

elzadj committed:
fix button tests

elzadj committed:
remove broken style

kyrapieterse committed:
Refine clip-path control styling and wrapper class

Added a wrapper class to the clip-path control for more targeted styling. Updated SCSS to reset inherited styles, adjust spacing, and remove padding from the accordion panel containing the clip-path control. Changed editor content background color for improved UI consistency.

kyrapieterse committed:
Update styles and add class to clip path mode control

Removed unnecessary margin from accordion-maxi styles. Added custom margin for tabs in clip-path mode control and applied a new class in the component. Updated active tab color in setting-tabs-control to use primary color.

kyrapieterse committed:
Refine background and tab control styles

Improves spacing and styling for background control, image layer, and tab controls. Adds unique classes for parallax and settings tabs, updates separator styling, and adjusts margins for better UI consistency. Also tweaks info box and scroll effects control styles for improved layout.

kyrapieterse committed:
Refactor advanced number control SCSS for clarity

Simplifies and restructures the SCSS for the advanced number control component by consolidating shared patterns, improving layout and accessibility styles, and enhancing maintainability. Removes redundant comments and streamlines selectors for better readability and future updates.

kyrapieterse committed:
Add margin to checked loop toggle switch

Introduces a 10px top margin for the checked state of the loop context toggle switch to improve spacing in the UI.

kyrapieterse committed:
Adjust spacing and alignment in control components

Updated SCSS styles for axis, base, and relation controls to refine vertical spacing and alignment. Margin-bottom values were reduced for tighter layout, and align-items was set to center for improved visual consistency.

kyrapieterse committed:
Refine UI styles and add buttonClasses to AxisPositionControl

Adjusted margins and paddings in various SCSS files for improved UI consistency. Added buttonClasses prop to AxisPositionControl and SearchMaxi inspector for more flexible button styling. Updated NavigationIconControl to use labels instead of icons for options.

kyrapieterse committed:
Update icon block color from orange to green

Changed the --maxi-icon-block-orange variable and related SVG icon fills from #ff4a17 (orange) to #2c8a46 (lime green) across CSS, SCSS, JS, and test files for consistency with new branding. Also adjusted margin styles in video-maxi editor for improved layout.

kyrapieterse committed:
Add margin adjustments for tab button elements

Introduces negative left and right margins for .maxi-tabs-control__button-title and .maxi-tabs-control__button-description in editor.scss to improve tab button layout. No functional changes in style.scss.

elzadj committed:
Fix colour preset tests

elzadj committed:
Tidy css and some js files

elzadj committed:
Fix a missing bracket in ANC

elzadj committed:
hide tag setting for search and button

elzadj committed:
hide tag setting for list item

elzadj committed:
Fix a typo

elzadj committed:
Changelog

elzadj committed:
generate pot file

elzadj committed:
Fix old video breaking because of the icons colour change

elzadj committed:
Chagne the date

elzadj committed:
Fix same-arogin for videos on frontend

elzadj committed:
Fix same-origin bug with frontend bg yt videos
